governance/polls/Reserve Governance Facilitator Apppointment - August 21, 2023.md

```diff
@@ -0,0 +1,80 @@
+  result_display: approval-breakdown
+options:
+   0: Abstain
+   1: Le Bateleur
+   2: VoteWizard
+   3: DAO Masons
+   4: JanSky
+   5: None of the above
+start_date: 2023-08-21T16:00:00
+end_date: 2023-08-24T16:00:00
+---
+# Poll: Reserve Governance Facilitator Apppointment - August 21, 2023
+
+The Governance Facilitators have placed a Governance Poll into the voting system using their powers defined in [MIP113.7.1.4](https://mips.makerdao.com/mips/details/MIP113#7-1-4-1). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, August 21 at 16:00 UTC.
+
+**This is an approval vote.**
+- **You may vote for multiple options.**
+- **You should vot for all options that you would support becoming a Reserve Governance Facilitator.**
+- **Unselected options signal that you do not approve of those options.**
+- **If you do not approve of any of the options then you should selecte "None of the above" as your one and only choice.**
+- **If you wish to abstain, that should be your one and only choice.**
+
+## Note on Display of Winners
+
+Due to limitations of the voting portal frontend, we cannot select multiple winners. However, if up to three candidates receive >50% approval, all of the candidates above 50% will be appointed as Reserve Facilitators, this will be confirmed off-chain in the MakerDAO forum.
+
+If more than three candidates receive >50% approval, the three candidates with the most votes will become Reserve Facilitators.
+
+## Review
+
+The community can vote in this poll to express approval for the following Reserve Governance Facilitator options:
+* Le Bateleur
+* VoteWizard
+* DAO Masons
+* JanSky
+* None of the above
+
+Please review the discussion [thread](https://forum.makerdao.com/t/poll-notification-selection-of-reserve-governance-facilitators/21671) to help inform your position before voting.
+
+In addition, evaluations of all candidates have been posted in this [thread](https://forum.makerdao.com/t/govalpha-succession-candidate-evaluations/21696).
+
+Each candidate has also posted an introductory post. For convenience, these are linked below.
+
+* [Le Bateleur](https://forum.makerdao.com/t/le-bateleur-govalpha-succession-candidate-introduction/21701)
+* [VoteWizard](https://forum.makerdao.com/t/votewizard-introduction-govalpha-handover/21700)
+* [DAO Masons](https://forum.makerdao.com/t/dao-masons-introduction-re-govalpha-handover/21699)
+* [JanSky](https://forum.makerdao.com/t/jansky-interim-governance-facilitator-candidate-statement/21712/1)
+
+## Outcomes
+
+**The winner(s) of this vote require(s) support of an overall majority of votes. The non-`Abstain` option(s) with the support of a majority of votes up to a total of three candidates will be deemed the winner(s).**
+**If no candidate receives a majority of votes, no Reserve Facilitator will be elected.**
```

Comment:
Or if none-of-the above receives a plurality?

Reply:
well that would be the same thing, can't have a plurality for none-of-the-above and someone with > 50% approval

Reply:
Actually, just change this to be: "Otherwise, no reserve facilitator will be elected"